Scripts/uetorch.lua

```diff
@@ -508,6 +510,7 @@ uetorch.SetActorVisible = utlib.SetActorVisible
 uetorch.SetActorVelocity = utlib.SetActorVelocity
 uetorch.SetActorAngularVelocity = utlib.SetActorAngularVelocity
 uetorch.SetActorScale3D = utlib.SetActorScale3D
+uetorch.DestroyActor = utlib.DestroyActor
```

Comment:
Yes, the problem is that calling ObjectSegmentation many times was slow unless the stride was increased. The constant just helps to make a difference between the black font and the material I'm using for the actor.